huaweicloud/services/cci/resource_huaweicloud_cci_namespace.go

```diff
@@ -0,0 +1,335 @@
+					validation.StringLenBetween(1, 63),
+				),
+			},
+			"auto_expend_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"enterprise_project_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"warmup_pool_size": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Computed:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.IntBetween(1, 500),
+			},
+			"recycling_interval": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Computed:     true,
+				ForceNew:     true,
+				RequiredWith: []string{"warmup_pool_size"},
+			},
+			"container_network_enabled": {
+				Type:         schema.TypeBool,
+				Optional:     true,
+				Computed:     true,
+				ForceNew:     true,
+				RequiredWith: []string{"warmup_pool_size"},
+			},
+			"rbac_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+				ForceNew: true,
+			},
+			"created_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func buildCciNamespaceCreateParams(d *schema.ResourceData, config *config.Config) (namespaces.CreateOpts,
+	error) {
+	createOpts := namespaces.CreateOpts{
+		Kind:       "Namespace",
+		ApiVersion: "v1",
+		Metadata: namespaces.Metadata{
+			Name: d.Get("name").(string),
+			Annotations: namespaces.Annotations{
+				Flavor:     d.Get("type").(string),
+				AutoExpend: d.Get("auto_expend_enabled").(bool),
+			},
+			Labels: &namespaces.Labels{
+				EnterpriseProjectID: config.GetEnterpriseProjectID(d),
+				RbacEnable:          d.Get("rbac_enabled").(bool),
+			},
+		},
+	}
+
+	if size, isAdvance := d.GetOk("warmup_pool_size"); isAdvance {
+		createOpts.Metadata.Annotations.PoolSize = size.(int)
+		createOpts.Metadata.Annotations.RecyclingInterval = d.Get("recycling_interval").(int)
+		if enabled, ok := d.GetOk("container_network_enabled"); ok && enabled.(bool) {
+			createOpts.Metadata.Annotations.NetworkEnable = "vpc-network-ready"
+		}
+	}
+
+	return createOpts, nil
+}
+
+func resourceCciNamespaceCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	config := meta.(*config.Config)
+	client, err := config.CciV1Client(config.GetRegion(d))
+	if err != nil {
+		return fmtp.DiagErrorf("Error creating HuaweiCloud CCI v1 client: %s", err)
+	}
+
+	createOpts, err := buildCciNamespaceCreateParams(d, config)
+	if err != nil {
+		return fmtp.DiagErrorf("Unable to build createOpts of the CCI namespace: %s", err)
+	}
+	ns := d.Get("name").(string)
+	namespace, err := namespaces.Create(client, createOpts).Extract()
+	if err != nil {
+		return fmtp.DiagErrorf("Error creating HuaweiCloud CCI PVC: %s", err)
+	}
+	d.SetId(namespace.Metadata.UID)
+	stateRef := stateRefresh{
+		Pending:      []string{"Pending", "Active"},
```

Comment:
"Active" is already in Target List, you can delete it

Reply:
sure